### PR TITLE
Fix navigation & context menu in Music Playlist Editor 

### PR DESCRIPTION
**Pull request by ronie**
## Description
This fixes two things in the Music Playlist Editor:
- navigating into the Files directory would display an incorrect listing
- the context menu on the right container was broken (it didn't show up at all)

## How Has This Been Tested?
Tested in Estuary by creating a new playlist and:
- adding items
- deleting items
- moving items up
- moving items down

## Screenshots (if appropriate):
Files directory (before):
files-b

Files directory (after):
files-a

Context menu:
context

## Types of change

- [X] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [ ] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:

- [X] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [X] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

File: xbmc/music/windows/GUIWindowMusicPlaylistEditor.cpp [hunk 1]
```diff
@@ -396,8 +372,19 @@ void CGUIWindowMusicPlaylistEditor::AppendToPlaylist(CFileItemList &newItems)
 void CGUIWindowMusicPlaylistEditor::OnPlaylistContext()
 {
   int item = GetCurrentPlaylistItem();
```
Comment (DaveTBlake) [inline xbmc/music/windows/GUIWindowMusicPlaylistEditor.cpp:374]: item is unused (and upsets Clang)

**Review by DaveTBlake — COMMENTED** [review]
Thanks @ronie, great to have the playlist editor useable from just TV remote again with context menus and not needing keyboard.

I was going in similar direction but you beat me to it. Small fix to make Clang happy which is currently failing the build - line 106
```int item = GetCurrentPlaylistItem();```
variable item is unused

**Comment by ronie** [comment]
good catch. i've removed that line as well now.

**Review by DaveTBlake — APPROVED** [review]

**Comment by DaveTBlake** [comment]
This PR fixes issue https://github.com/xbmc/xbmc/issues/16793

**Pull request merged**